schemas/2019-05-01/Microsoft.ContainerRegistry.json

```diff
@@ -0,0 +1,768 @@
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/common/definitions.json#/definitions/expression"
+            }
+          ],
+          "description": "The tags for the webhook."
+        },
+        "location": {
+          "type": "string",
+          "description": "The location of the webhook. This cannot be changed after the resource is created."
+        },
+        "properties": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/WebhookPropertiesCreateParameters"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/common/definitions.json#/definitions/expression"
+            }
+          ],
+          "description": "The parameters for creating the properties of a webhook."
+        }
+      },
+      "required": [
+        "name",
+        "type",
+        "apiVersion",
+        "location",
+        "properties"
+      ],
+      "description": "Microsoft.ContainerRegistry/registries/webhooks"
+    },
+    "RegistryProperties": {
+      "type": "object",
+      "properties": {
+        "adminUserEnabled": {
+          "oneOf": [
+            {
+              "type": "boolean",
+              "default": false
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/common/definitions.json#/definitions/expression"
+            }
+          ],
+          "description": "The value that indicates whether the admin user is enabled."
+        },
+        "storageAccount": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/StorageAccountProperties"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/common/definitions.json#/definitions/expression"
+            }
+          ],
+          "description": "The properties of a storage account for a container registry. Only applicable to Classic SKU."
+        },
+        "networkRuleSet": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/NetworkRuleSet"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/common/definitions.json#/definitions/expression"
+            }
+          ],
+          "description": "The network rule set for a container registry."
+        },
+        "policies": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/Policies"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/common/definitions.json#/definitions/expression"
+            }
+          ],
+          "description": "The policies for a container registry."
+        }
+      },
+      "description": "The properties of a container registry."
+    },
+    "ReplicationProperties": {
+      "type": "object",
+      "properties": {},
+      "description": "The properties of a replication."
+    },
+    "RetentionPolicy": {
```

Comment:
Missing property `lastUpdatedTime` 
[Here ](https://github.com/Azure/azure-rest-api-specs/blob/daa4465509ed3d43260a7d1bfe7e2ca0b70ac9b9/specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2019-05-01/containerregistry.json#L1627)is the property is current swagger:

Reply:
Note that `readOnly` properties are removed intentionally - they cannot be directly set with a resource PUT during template deployment.